development/vulnerabilities-scanner.sh

```diff
@@ -105,20 +105,24 @@ function testComponents() {
   do
 
     echo "processing ${DIR}"
-
+    
     GOPKG_FILE_NAME="${DIR}"Gopkg.lock
 
     if [ -f "${GOPKG_FILE_NAME}" ]; then
+      set +e
```

Comment:
`dep ensure || dep ensure` would be a simple way to achieve the same result with setting that option.